requirements.txt

```diff
@@ -1,5 +1,6 @@
 # core deps
 boto==2.25.0
+boto3=1.3.1
```

Comment:
definitely agreed. fixed in my branch. I'm uncertain if/how to propagate that here.